src/Card.js

```diff
@@ -2,49 +2,45 @@ import styled from 'styled-components'
 import Box from './Box'
 import PropTypes from 'prop-types'
 
-const boxShadow = (props) => {
-
+const boxShadow = props => {
   const boxShadows = {
-    'sm': {
+    sm: {
       'box-shadow': props.theme.boxShadows[0]
     },
-    'md': {
-      'box-shadow': props.theme.boxShadows.slice(0,2).join(', ')
+    md: {
+      'box-shadow': props.theme.boxShadows.slice(0, 2).join(', ')
     },
-    'lg': {
-      'box-shadow': props.theme.boxShadows.slice(0,3).join(', ')
+    lg: {
+      'box-shadow': props.theme.boxShadows.slice(0, 3).join(', ')
     },
-    'xl': {
+    xl: {
       'box-shadow': props.theme.boxShadows.join(', ')
     }
   }
   return boxShadows[props.boxShadowSize]
 }
 
-const boxBorder = (props) => {
+function boxBorder(props) {
```

Comment:
Did prettier do this as well?

Reply:
Hm, that one seems pretty invasive.  I think prettier did, in fact, do this.

Reply:
Guessing it's because of the `return`